### ISSUE TITLE
Adding products inventory/register

### ISSUE THREAD

**Issue by kkoci** [maxolasersquad/orthosie#15]
I'm on http://127.0.0.1:8000/inventory/

No matetr if I'm on UPC or PLU, I just can't add anything =/, there's no button, am I missing something?

**Comment by maxolasersquad** [comment]
After typing in the PLU/UPC press the enter key.

**Comment by kkoci** [comment]
Hi, yes tab works, but Enter doesn't it says 'Invalid UPC'

**Comment by maxolasersquad** [comment]
I just tried and did not have any problem adding a new Inventory item. What was the "upc" that was giving you problems?

**Comment by kkoci** [comment]
Actually none, I was just trying to add one from /inventory url, if I try to add a PLU throws a server request error, but it registers it anyways, uh... I'm using chrome

**Comment by kkoci** [comment]
Also, on /register there are some errors, can't figure out how it really works, gonna try and add some features asap

**Comment by maxolasersquad** [comment]
I am completely rewriting this part of the system right now to use a REST api. I'll check back with you when that's done.

**Comment by kkoci** [comment]
Awesome! Really great, I'm thinking about adding some features myself, I'm working on it by my side, we can check it out later too, cheers!

**Comment by maxolasersquad** [comment]
That's great. Before you code anything put in a bug/feature request so we
can discuss. I know I've built features for projects before that ultimately
got rejects which is rather disheartening. I am currently working on moving
the interface to handling all of its interactions via a restful api.

**Comment by kkoci** [comment]
Yes, it is disheartening for sure, Okay, I'll do it that way then, Thank You

**Comment by jaustinlam** [comment]
I am having a similar issue. I can add an item through a PLU but adding through a UPC gives me:
`AssertionError: ("Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the GrocerySerializer serializer.",`
